python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_conditional_expectation.py

```diff
@@ -0,0 +1,669 @@
+# of input variables, we want to create a new function :math:`\widehat{\model}`:
+#
+# .. math::
+#     \widehat{\model}: \Rset^{|\vect{u}|} \rightarrow \Rset
+#
+# where :math:`|\vect{u}| = \operatorname{card}(\vect{u})` is the
+# number of variables in the group.
+#
+# In this example, we experiment two different ways to reduce the
+# input dimension of a polynomial chaos expansion:
+#
+# - the parametric function,
+# - the conditional expectation.
+#
+# The goal of this page is to see how we can create these
+# functions and the difference between them.
+
+# %%
+# Parametric function
+# ~~~~~~~~~~~~~~~~~~~
+#
+# The simplest method to reduce the dimension of the input
+# is to set some input variables to constant values.
+# In this example, all marginal inputs, except those in
+# the conditioning indices are set to the mean of the input random vector.
+#
+# Let :math:`\overline{\vect{u}}` be the complementary set of input
+# marginal indices such that :math:`\vect{u}` and :math:`\overline{\vect{u}}`
+# form a disjoint partition of the full set of variable indices:
+#
+# .. math::
+#
+#     \vect{u} \; \dot{\cup} \; \overline{\vect{u}} = \{1, ..., \physicalInputDimension\}.
+#
+# The parametric function with reduced dimension is:
+#
+# .. math::
+#
+#   \widehat{\model}(\physicalInputObservation_{\vect{u}})
+#   = \model\left(\physicalInputObservation_{\vect{u}},
+#            \physicalInputObservation_{\overline{\vect{u}}}
+#            = \Expect{\physicalInputRandomVector_{\overline{\vect{u}}}}\right)
+#
+# for any :math:`\physicalInputObservation_{\vect{u}} \in \Rset^{|\vect{u}|}`.
+# The previous function is a parametric function based on the function :math:`\model`
+# where the parameter is :math:`\Expect{\physicalInputRandomVector_{\overline{\vect{u}}}}`.
+# Assuming that the input random vector has an independent copula,
+# computing :math:`\Expect{\physicalInputRandomVector_{\overline{\vect{u}}}}`
+# can be done by selecting the corresponding indices in :math:`\Expect{\physicalInputRandomVector}`.
+# This function can be created using the :class:`~openturns.ParametricFunction`
+# class.
+
+# %%
+# Parametric PCE
+# ~~~~~~~~~~~~~~
+#
+# If the physical model is a PCE, then the associated parametric model is also a
+# PCE.
+# Its coefficients and the associated functional basis can be computed from
+# the original PCE.
+# A significant fact, however, is that the coefficients of the parametric
+# PCE are *not* the ones of the original PCE: the coefficients of the parametric
+# PCE have to be multiplied by factors which depend on the
+# value of the discarded basis functions on the parameter vector.
+# This feature is not currently available in the library.
+# However, we present it below as this derivation is interesting
+# to understand why the conditional expectation may behave
+# differently from the corresponding parametric PCE.
+
+# %%
+# Let :math:`h` be the PCE in the standard space:
```

Comment:
```suggestion
```suggestion
# Let :math:`\cJ^P \subseteq \Nset^{\physicalInputDimension}` be the set of
# multi-indices corresponding to the truncated polynomial chaos expansion
# up to the :math:`P`-th coefficient.
# Let :math:`h` be the PCE in the standard space:
```
```